README.md

```diff
@@ -336,12 +336,9 @@ export abstract class AbstractTestBase {
 }
 ```
 
-One can also inherit from another concrete test class or suite, but keep in mind that that the tests declared by that suite will
-be run multiple times and not just in the context of the concrete test classes, see the example provided in `tests/ts/suite.inheritance.suite.ts`.
+Note: You can override test methods inherited from a base class and then call `super()` in order to run the assertions implemented by the super class.
 
-Important: One should not override test methods inherited from a base class and then call `super()` as this will run the tests from the base class twice.
-
-Best practice: Do not inherit from other classes that have been decorated with the ``suite`` decorator. Instead use abstract test base classes.
+Best practice: You must not inherit from other classes that have been decorated with the ``suite`` decorator. Instead use abstract test base classes. Doing so will result in an exception.
```

Comment:
Reword "Doing so", it is close to the second sentence where the positive case is described. Maybe replace with "Inheriting with classes decorated with ``suite`` will result in an exception."

Reply:
will do